src/ansys/fluent/core/filereader/case.py

```diff
@@ -3,16 +3,13 @@
 Example
 -------
 
-from ansys.fluent.core.filereader.case import Case
+.. code-block:: python
 
-Instantiate a case reader
+    from ansys.fluent.core.filereader.case import Case
```

Comment:
@hpohekar: @prmukherj convinced me that we should call the class `CaseFile`

Reply:
https://github.com/pyansys/pyfluent/pull/1486, this PR is waiting to be merged after a bit of discussion which can be done in today's meeting? @seanpearsonuk, @hpohekar